### PR TITLE
Test for compareReducer

### PR DESCRIPTION
**Pull request by johnsonm325**

### CODE REVIEW

File: 1 [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
```
Comment (ehelms) [inline 1]: This file looks unintentional to include?

File: src/SmartComponents/modules/__tests__/reducer-test.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+                        last_updated: '2019-01-15T14:53:15.886891Z'
+                    },
+                    {
+                        fdqn: 'ibm-x3650m4-03-vm03.lab.eng.brq.redhat.com',
+                        id: 'f35b1e1d-d231-43f2-8e4f-8f9cb01e3aa2',
+                        last_updated: '2019-01-15T15:25:16.304899Z'
+                    }
+                ],
+                /* eslint-enable camelcase */
+                type: types.FETCH_COMPARE
```
Comment (ehelms) [inline src/SmartComponents/modules/__tests__/reducer-test.js]: I think this should be `${types.FETCH_COMPARE}_FULFILLED` given thats the action in the reducer.

**Comment by ehelms** [comment]
Looks like it thinks your fixture file should be a test file and is throwing an error.

**Comment by johnsonm325** [comment]
> Looks like it thinks your fixture file should be a test file and is throwing an error.

Any tips on  how to ignore files would be greatly appreciated. :)

**Comment by ehelms** [comment]
In package.json `jest` section, try adding `testPathIgnorePatterns` and include a pattern for 'fixtures'

**Pull request merged**